plextraktsync/sync.py

```diff
@@ -153,15 +153,16 @@ def sync(self, walker: Walker, dry_run=False):
             for show in walker.walk_shows(shows, title="Syncing show ratings"):
                 self.sync_ratings(show, dry_run=dry_run)
 
-        if self.sync_wl or self.config.sync_liked_lists:
+        if self.config.update_plex_wl_as_pl or self.config.sync_liked_lists:
             if is_partial:
-                logger.warning("Partial walk, watchlist and/or liked list updating was skipped")
+                logger.warning("Partial walk, liked list updating was skipped")
```

Comment:
To be consistent, reformulate to 
`logger.warning("Running partial library sync. Liked lists won't update because it needs full library sync.")`

Reply:
if you use github code suggestions would be a lot better to accept changes!

```suggestion
                logger.warning("Running partial library sync. Liked lists won't update because it needs full library sync.")
```